bot/commands/mute.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../../config.json');
+const Discord = require('discord.js');
+
+module.exports.execute = async (client, message, args) => {
+    if (message.member.roles.cache.has(config.bot.roles.staff)) {
+        if (args.length == 0) { await message.channel.send('Mention someone to mute!') } else {
+            let role = message.guild.roles.cache.get(config.bot.roles.muted)
+            let user = message.mentions.members.first()
```

Comment:
The command fails if a user has not been mentioned. Maybe add a try catch?

Reply:
Alright everything should be fixed